README.md

```diff
@@ -17,7 +17,7 @@ To integrate with Yoti IDV, a working infrastructure is needed (see [developers.
 Start your integration by adding the following dependency to your `package.json` file:
 ```json
 "dependencies": {
-    "@getyoti/yoti-doc-scan-react-native": "^3.0.0"
+    "@getyoti/yoti-doc-scan-react-native": "^3.0.2"
```

Comment:
should revert this

Reply:
Done: 91ce3f17d902f6a87e00bc33f70da3b800f6db9b 👍